kolibri/core/assets/src/views/app-bar.vue

```diff
@@ -47,6 +59,46 @@
           <div v-else-if="isCoach">{{ $tr('coach') }}</div>
           <div v-else-if="isLearner">{{ $tr('learner') }}</div>
         </template>
+
+        <template slot="options">
+          <template v-if="isUserLoggedIn">
+            <menu-option
+              :label="$tr('profile')"
+              @select="selectOption($tr('profile'))"
```

Comment:
I could have emitted the label?

Reply:
It's a little weird to use the `$tr` string below to drive logic, e.g. `option === this.$tr('profile')`.

Instead, I'd recommend creating a unique method for each possible action, e.g. `@select="selectProfile"`, `@select="selectLanguage"`, etc

Reply:
I totally agree, I was just trying to reuse the function. :man_facepalming: